### DISCUSSION TITLE
New with v8:   forecast module goes open source + license change from AGPL to MIT

### DISCUSSION THREAD

**Discussion by jdetaeye** [frePPLe/frepple#588] [Announcements]
This release brings two exciting changes to the open source Community Edition:

1. The forecasting module is moving into the Community Edition. So far, it was available only in the Enterprise and Cloud Editions.
    Read [this post](https://frepple.com/blog/the-forecast-module-goes-open-source/) to learn more.

2. The license of the Community Edition changes from [AGPL](https://wikipedia.org/wiki/Affero_General_Public_License) to [MIT](https://wikipedia.org/wiki/MIT_License).
Read [this post](https://frepple.com/blog/why-we-are-changing-our-license-from-agpl-to-mit/) to learn more.

Join the [webinar on Thursday June 8 at 4 PM Central European Time](https://frepple.com/webinar_frepple_8.ics) to hear the full story and ask us any questions you may have.

Enjoy!

The frePPLe team

**Comment by matmair** [comment]
Very cool news. Do you have an official branding package that can be used? An SVG logo, an official tagline and a short description? Would love to include a link to the software in an article now that it is permissively licensed.

**Reply by jdetaeye** [reply]
We'll work out some official branding guidelines shortly, and I'll keep you posted.
You find the logo at https://raw.githubusercontent.com/frePPLe/frepple/master/freppledb/common/static/img/frepple.svg

**Reply by jdetaeye** [reply]
@matmair Here's the branding guidelines I just put together    https://frepple.com/docs/current/branding-guidelines.html    As with anything we do: all feedback welcome!

**Reply by matmair** [reply]
Thank you @jdetaeye ! I will make sure to include a link to https://frepple.com/

**Comment by onepiecewyz** [comment]
Awesome，Frepple's development is going to accelerate!
